### PR TITLE
Dts 20266 clean field mapping fields

### PR DESCRIPTION
**Pull request by HinPublicis**

### CODE REVIEW

File: processors/azure/azure-boards/src/test/java/com/publicissapient/kpidashboard/azure/client/azureissue/ScrumAzureIssueClientImplTest.java [hunk 1]
```diff
@@ -397,8 +393,6 @@ private void prepareFieldMapping() {
 		jiraSegData.add("Ready for Testing");
 		fieldMapping.setJiraStatusForQa(jiraSegData);
 		fieldMapping.setDevicePlatform("customfield_18181");
```
Comment (hirbabar) [inline processors/azure/azure-boards/src/test/java/com/publicissapient/kpidashboard/azure/client/azureissue/ScrumAzureIssueClientImplTest.java:395]: need to remove

File: processors/jiratest/src/test/java/com/publicissapient/kpidashboard/jiratest/processor/service/impl/JiraTestServiceImplTest.java [hunk 2]
```diff
@@ -208,7 +208,6 @@ private static ProcessorToolConnection getJiraToolConfig() {
 		String[] testCaseType = new String[2];
 		testCaseType[0] = "Test";
 		testCaseType[1] = "TestCase";
-		toolInfo.setJiraTestCaseType(testCaseType);
 		toolInfo.setTestAutomatedIdentification("CustomField");
```
Comment (hirbabar) [inline processors/jiratest/src/test/java/com/publicissapient/kpidashboard/jiratest/processor/service/impl/JiraTestServiceImplTest.java:211]: revert back

**Review by hirbabar — APPROVED** [review]

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/projectconfig/projecttoolconfig/service/ProjectToolConfigServiceImpl.java [hunk 3]
```diff
@@ -224,7 +224,6 @@ public ServiceResponse modifyProjectToolById(ProjectToolConfig projectToolConfig
 		projectTool.setRegressionAutomationFolderPath(projectToolConfig.getRegressionAutomationFolderPath());
 		projectTool.setInSprintAutomationFolderPath(projectToolConfig.getInSprintAutomationFolderPath());
 		projectTool.setOrganizationKey(projectToolConfig.getOrganizationKey());
-		projectTool.setJiraTestCaseType(projectToolConfig.getJiraTestCaseType());
```
[the remark this answers is not in the document]
Comment (laksingh3) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/projectconfig/projecttoolconfig/service/ProjectToolConfigServiceImpl.java:227]: Done

[the remark this answers is not in the document]
Comment (hirbabar) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/projectconfig/projecttoolconfig/service/ProjectToolConfigServiceImpl.java:227]: needed to revert still

**Review by Chittauri — APPROVED** [review]

**Pull request merged**